runtime/intern.c

```diff
@@ -95,8 +95,14 @@ struct caml_intern_state {
   /* 1 if the compressed format is in use, 0 otherwise */
 };
 
-/* Allocates the domain local intern state if needed */
-static struct caml_intern_state* get_intern_state (void)
+static void intern_init_stack(struct caml_intern_state* s)
+{
+  /* (Re)initialize the globals for next time around */
+  s->intern_stack = s->intern_stack_init;
+  s->intern_stack_limit = s->intern_stack + INTERN_STACK_INIT_SIZE;
+}
+
+static struct caml_intern_state* prepare_intern_state (void)
```

Comment:
`init_intern_state` is a better name.

Reply:
Ok. The name was taken from `extern.c`. For consistency, I've also made the change there.